technical-advisory-groups/security-tag/README.md

```diff
@@ -0,0 +1,21 @@
+# Security TAG
+
+## Mission Statement
+The OpenSearch Security Technical Advisory Group (TAG) champions the strategic direction and technical guidance for security solutions within the OpenSearch ecosystem.
+
+[Charter](./charter.md)
+
+### Membership
+
+We follow the membership guidelines proposed in [TAGs README](../README.md) to add / remove / amend / retire members for Security TAG.
+
+| Member            | GitHub ID                                                    | Affiliation |
+| ----------------- | ------------------------------------------------------------ | ----------- |
+| Craig Perkins     | [@cwperks](https://github.com/cwperks)                       | Amazon      |
+| Nils Bandener     | [@nibix](https://github.com/nibix)                           | Eliatra     |
+| Aparajita Pandey  | [@aparajita31pandey](https://github.com/aparajita31pandey)   | Uber        |
+| Gulshan Kumar     | [@kumargu](https://github.com/kumargu)                       | Amazon      |
+| Kunal Khatua      | [@kkhatua](https://github.com/kkhatua)                       | Amazon      |
+| Andrey Pleskach   | [@willyborankin](https://github.com/willyborankin)           | Aiven       |
+| Shikhar Jain      | [@shikharj05](https://github.com/shikharj05)                 | Amazon      |
+| Jochen Kressin    | [@jochen-kressin](https://github.com/jochen-kressin)         | Eliatra     |
```

Comment:
If possible, can we have the member email in a new column?

One issue we see in other members list is that overtime it is hard to contact people if they are not active on github anymore. 

Thanks.

Reply:
ack will collect and add a column